### ISSUE TITLE
File not found error while running start script

### ISSUE THREAD

**Issue by Prathamesh-B** [codegods/blogit#13]
**Description**

start script fails with the following error:

```
[14:42:00.391721] [INFO] configLoader: Loading config from D:\Coding\blogit\config.py
[14:42:00.401634] [INFO] configLoader: Validating configuration...
[14:42:00.428622] [INFO] configLoader: Configuration OK
[14:42:00.432615] [INFO] startScript: Using npm package manager...
[14:42:00.433614] [INFO] startScript: Attempting to start webpack dev server...
Traceback (most recent call last):
  File "d:\Coding\blogit\scripts\start.py", line 195, in <module>
    main()
  File "d:\Coding\blogit\scripts\start.py", line 183, in main
    subprocess.Popen(
  File "C:\Users\Prathamesh1704\AppData\Local\Programs\Python\Python38\lib\subprocess.py", line 854, in __init__
    self._execute_child(args, executable, preexec_fn, close_fds,
  File "C:\Users\Prathamesh1704\AppData\Local\Programs\Python\Python38\lib\subprocess.py", line 1307, in _execute_child
    hp, ht, pid, tid = _winapi.CreateProcess(executable, args,
FileNotFoundError: [WinError 2] The system cannot find the file specified
```

**Steps to reproduce**
Run `python -m scripts.start` in a system

**Operating system:**
> Windows 10

**Comment by jalaj-k** [comment]
I have tried to fix it in dc22a75. But it is just a workaround as using `shell=True` is a security hazard [ [Reference] ](https://docs.python.org/3/library/subprocess.html#security-considerations). I am working on another solution.

**Comment by jalaj-k** [comment]
Fixed in 5f8eba1.
